### DISCUSSION TITLE
How can I remove and reinstall the app before launching tests

### DISCUSSION THREAD

**Discussion by gxs-antran** [mobile-dev-inc/Maestro#583] [Q&A] [answered]
For some flows, I need to test under the condition of a fresh install. I can do that for sure via scripts, but is there any step in Maestro DSL that I can do it inside the flow yaml?

**Comment by axelniklasson** [comment, marked as answer]
Hi @gxs-antran! There is no reinstallation command in maestro but you can use `clearState` along with `launchApp` to clear the app state - [docs here](https://maestro.mobile.dev/api-reference/commands/launchapp). If that doesn't work for you, please file a feature request [here](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BFeature+Request%5D). Thanks!
